src/FormulateInputGroup.vue

```diff
@@ -54,7 +54,9 @@ export default {
   methods: {
     groupItemContext (context, option, groupAttributes) {
       const optionAttributes = {}
-      const ctx = Object.assign({}, context, option, groupAttributes, optionAttributes)
+      const ctx = Object.assign({}, context, option, groupAttributes, optionAttributes, !context.hasGivenName ? {
+        name: true
+      } : {})
```

Comment:
@andrew-boyd Here's what I had in mind to solve the issue for radios and checkboxes.

Reply:
Ah, much more elegant. Thank you.